### ISSUE TITLE
[as designed?] errors.txt not deleted after errors are fixed

### ISSUE THREAD

**Issue by joelrbrandt** [adobe-photoshop/generator-assets#132]
Both before and after #130, it seems that errors.txt lives around even after errors are fixed.

Is this as designed (i.e. is it supposed to be viewed as a log file)? Or is the long-term intent to remove this document after it is no longer relevant?
